hack/test-cmd.sh

```diff
@@ -98,6 +91,53 @@ export OPENSHIFT_PROFILE="${CLI_PROFILE-}"
 # Begin tests
 #
 
+# test client not configured
+[ "$(osc get services 2>&1 | grep 'OpenShift is not configured')" ]
```

Comment:
Is this error going to be returned if I pass a config parameter that sets the value?